firebase-firestore/src/test/java/com/google/firebase/firestore/spec/SpecTestCase.java

```diff
@@ -501,6 +506,20 @@ private void doUnlisten(JSONArray unlistenSpec) throws Exception {
     queue.runSync(() -> eventManager.removeQueryListener(listener));
   }
 
+  private void doLoadBundle(String json) throws Exception {
```

Comment:
This confuses me, how did spec tests run before this PR?

Reply:
They have "no-android" tags and didn't run before they were removed in this PR.

Reply:
I see, thanks.